.github/workflows/cron-staging.yml

```diff
@@ -36,11 +36,12 @@ jobs:
             ${{ runner.os }}-${{ matrix.python-version }}
       - name: Install Deps
         run: python -m pip install -U tox setuptools virtualenv wheel
-      - name: Install and Run Tests
+      - name: Install and Run Tests (Windows and Linux)
         run: tox -e terra-main
         if: runner.os != 'macOS'
-      - name: Install and Run Tests
+      - name: Install and Run Tests (Macs only)
         run: tox -e terra-main
         if: runner.os == 'macOS'
         env:
+          TEST_TIMEOUT: 120
```

Comment:
Have the cron tests been more reliable than the main tests? I don't see the OMP_NUM_THREADS setting in the main file and that's the kind of parallelization disabling that might help with interfering parallel tests.

Reply:
Ah, just left a comment about the same thing.

Reply:
Nice! If we had time, it might be good to look at this kind of thing more closely. Even though Windows and Linux are not timing out, they might still benefit from disabling parallelization of math operations like in numpy since stestr is already trying to use one cpu per test. We would need to look up the right environment variables and compare the run times.